src/Instrumentation/PDO/src/PDOInstrumentation.php

```diff
@@ -31,14 +30,21 @@ public static function register(): void
                 $builder = self::makeBuilder($instrumentation, 'PDO::__construct', $function, $class, $filename, $lineno)
                     ->setSpanKind(SpanKind::KIND_CLIENT)
                     ->setAttribute(TraceAttributes::DB_CONNECTION_STRING, $params[0] ?? 'unknown')
-                    ->setAttribute(TraceAttributes::DB_USER, $params[1] ?? 'unknown')
-                    ->setAttribute(TraceAttributes::DB_SYSTEM, self::getDB($params[0]));
+                    ->setAttribute(TraceAttributes::DB_USER, $params[1] ?? 'unknown');
                 $parent = Context::getCurrent();
                 $span = $builder->startSpan();
                 Context::storage()->attach($span->storeInContext($parent));
             },
             post: static function (\PDO $pdo, array $params, mixed $statement, ?Throwable $exception) {
-                self::end($exception);            }
+                $scope = Context::storage()->scope();
+                if (!$scope) {
+                    return;
+                }
+                $span = Span::fromContext($scope->context());
+                $scheme = $pdo->getAttribute(\PDO::ATTR_DRIVER_NAME);
```

Comment:
could be a better variable name, like `$system` ?

Reply:
maybe `$dbSystem`, will be more meaningful